css/style.css

```diff
@@ -0,0 +1,545 @@
+  }
+}
+
+.back-side:hover {
+  animation: .7s rotation both;
+}
+
+@media (max-width: 1260px) {
+  .table-2 {
+    width: 1000px;
+    height: 620px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 180px;
+    height: 282px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 180px;
+    height: 282px;
+  }
+}
+
+@media (max-width: 1040px) {
+  .table-2 {
+    width: 850px;
+    height: 520px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 155px;
+    height: 242px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 155px;
+    height: 242px;
+  }
+}
+
+@media (max-width: 980px) {
+  .first-card,
+  .second-card {
+    width: 160px;
+  }
+
+  .first-card {
+    left: 55%;
+  }
+
+  .second-card {
+    left: 65%;
+  }
+}
+
+@media (max-width: 890px) {
+  .table-2 {
+    width: 680px;
+    height: 420px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 120px;
+    height: 188px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 120px;
+    height: 188px;
+  }
+
+  .level:hover::before {
+    display: none;
+  }
+}
+
+@media (max-width: 768px) {
+  .table-0 {
+    width: 580px;
+  }
+
+  .table-1 {
+    width: 580px;
+    height: 600px;
+  }
+
+  .card-wrapper {
+    width: 180px;
+    height: 282px;
+  }
+
+  .front-side,
+  .back-side {
+    width: 180px;
+    height: 282px;
+  }
+
+  .table-2 {
+    width: 175px;
+    height: 675px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 80px;
+    height: 125px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 80px;
+    height: 125px;
+  }
+}
+
+@media (max-width: 620px) {
+  .menu {
+    max-width: 92%;
+    text-align: center;
+  }
+
+  .first-card,
+  .second-card {
+    display: none;
+  }
+
+  .level {
+    margin-left: 0;
+  }
+
+  .lvl-active::after {
+    right: 77%;
+  }
+
+  .play {
+    display: block;
+    margin: 40px auto;
+  }
+
+  .level-choice__header {
+    font-size: 48px;
+  }
+
+  .level-choice__instruction {
+    margin-bottom: 100px;
+    font-size: 20px;
+  }
+
+  .front-side,
+  .back-side {
+    width: 155px;
+    height: 242px;
+  }
+
+  .card-wrapper {
+    width: 155px;
+    height: 242px;
+  }
+
+  .table-0 {
+    width: 500px;
+  }
+
+  .table-1 {
+    width: 500px;
+    height: 520px;
+  }
+
+  .table-2 {
+    width: 175px;
+    height: 675px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 80px;
+    height: 125px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 80px;
+    height: 125px;
+  }
+}
+
+@media (max-width: 540px) {
+  .level {
+    margin-left: 0;
+  }
+
+  .lvl-active::after {
+    right: 82%;
+  }
+
+  .play {
+    min-width: 250px;
+  }
+
+  .level-choice__header {
+    font-size: 40px;
+  }
+
+  .level-choice__instruction {
+    margin-bottom: 70px;
+  }
+
+  .front-side,
+  .back-side {
+    width: 120px;
+    height: 188px;
+  }
+
+  .card-wrapper {
+    width: 120px;
+    height: 188px;
+  }
+
+  .table-0 {
+    width: 400px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)